src/app/static/src/tests/localStorageManager.test.ts

```diff
@@ -114,4 +114,12 @@ describe("LocalStorageManager", () => {
         expect(spy.mock.calls[0][0]).toBe(`hintAppState_v${currentHintVersion}`);
         expect(spy.mock.calls[0][1]).toBe(JSON.stringify(testState));
     });
+
+    it("can save and get continue as guest from session", () => {
+        sessionStorage.setItem("asGuest", "continueAsGuest")
+        let result = sessionStorage.getItem("asGuest")
+
+        expect(result).not.toBe(null);
+        expect(sessionStorage.getItem("asGuest")).toBe("continueAsGuest");
+    });
```

Comment:
I can't see much value in this test any more, and certainly not in this file - it's just testing basic browser functionality rather than anything specific to `localStorageManager`